frame/recovery/src/lib.rs

```diff
@@ -262,22 +262,22 @@ pub mod pallet {
 	#[pallet::event]
 	#[pallet::generate_deposit(pub(super) fn deposit_event)]
 	pub enum Event<T: Config> {
-		/// A recovery process has been set up for an \[account\].
-		RecoveryCreated(T::AccountId),
+		/// A recovery process has been set up for an account.
+		RecoveryCreated { lost_account: T::AccountId },
```

Comment:
the name `account` or `who` might be better than `lost_account`.
When somebody create a recovery, it is not lost yet, it is just configured to be recoverable when it will be lost.